packages/esm-reports-app/src/components/simple-cron-editor/simple-cron-editor.component.tsx

```diff
@@ -202,26 +202,26 @@ const SimpleCronEditor: React.FC<SimpleCronEditorProps> = ({ initialCron, onChan
       setCron(editorState.cron);
     } else if (selectedScheduleType === ST_ONCE) {
       setCron(
-        `0 ${selectedTime.minutes} ${selectedTime.hours} ${editorState.date.getUTCDate()} ${
-          editorState.date.getUTCMonth() + 1
-        } ? ${editorState.date.getUTCFullYear()}`,
+        `0 ${selectedTime.minutes} ${selectedTime.hours} ${editorState.date.getDate()} ${
```

Comment:
I restored it, because date selected from datepicker is in local time zone e.g. 2025-02-13 00:00 CEST but in UTC this date is 2025-02-12 23:00, so better is just to keep the original date